index.html

```diff
@@ -642,12 +646,25 @@
       </p>
       <pre class="idl">
         interface PushManager {
+          [SameObject] static readonly attribute FrozenArray&lt;DOMString&gt; supportedContentEncodings;
```

Comment:
Agreed - the developer is exposed to the Content-Encoding header more than the fact it accepts a (list of) content coding.